include/swift/RemoteInspection/TypeRefBuilder.h

```diff
@@ -486,6 +455,368 @@ class TypeRefBuilder {
     }
   };
 
+  struct ReflectionTypeDescriptorFinder
+      : public swift::reflection::DescriptorFinder {
+    ReflectionTypeDescriptorFinder(TypeRefBuilder &Builder,
+                                   remote::ExternalTypeRefCache *externalCache)
```

Comment:
comment what externalCache is supposed to be?

Reply:
That's not related to this patch, but added a comment in the ivar explaining what that is.